xrootd/server/fshandler.go

```diff
@@ -27,9 +26,9 @@ import (
 	"go-hep.org/x/hep/xrootd/xrdproto/xrdclose"
 )
 
-// handler implements server.Handler API by making request to the backing filesystem at basePath.
-type handler struct {
-	server.Handler
+// fshandler implements server.Handler API by making request to the backing filesystem at basePath.
+type fshandler struct {
+	Handler
```

Comment:
is embedding `Handler` really needed?
(well... this can be part of another PR.)

Reply:
It allows to not bother about handling handshake / login / protocol requests here.

Reply:
ok.